src/repl.ml

```diff
@@ -148,7 +148,7 @@ let get_code text_editor =
     Vscode.TextDocument.getText document ~range:(selection :> Vscode.Range.t) ()
 
 let prepare_code code =
-  if String.is_suffix code ~suffix:";;" then code else code ^ ";;"
+  if String.is_suffix (String.trim code) ~suffix:";;" then code else code ^ ";;"
```

Comment:
We use `base` as a replacement for standard lib, so the following should be used instead:
```suggestion
  if String.is_suffix (String.strip code) ~suffix:";;" then code else code ^ ";;"
```